scripts/install/opt/seagate/cortx/motr/common/m0_utils_common.sh

```diff
@@ -465,17 +471,38 @@ run()
         eval "$*"
 }
 
+set_service_type()
+{
+        [[ "$1" == "DIX" ]] && SERVICE_TYPE=1 || SERVICE_TYPE=0
+}
+
+# input parameters:
+# (i) File name with path for conf.cg
+generate_conf_cg()
+{
+        local conf_cg_file=$1
+
+        CONF_FILE=$(find / -name confd.xc | head -n 1)
```

Comment:
Hey!! Please note, this will search from the top level "/" and down to all files.
This will take a lot of time.

This should be changed to only search a predefined path.

Reply:
Changed the command to search in specific directories.